src/main/java/net/mcreator/ui/minecraft/villagers/JVillagerTradeProfession.java

```diff
@@ -110,7 +110,7 @@ public JVillagerTradeProfession(MCreator mcreator, IHelpContext gui, JPanel pare
 	}
 
 	public void reloadDataLists() {
-		ComboBoxUtil.updateComboBoxContents(villagerProfession, ElementUtil.loadAllVillagerProfessions());
+		ComboBoxUtil.updateComboBoxContents(villagerProfession, ElementUtil.loadAllVillagerProfessions(mcreator.getWorkspace()));
```

Comment:
I believe the `workspace` field can be used here instead.

Reply:
Done!